src/ir/expr.rs

```diff
@@ -14,10 +14,20 @@ pub enum Expr {
     },
     Fn {
         op: ast::UnFn,
-        args: Box<[ExprIdx]>,
+        args: Box<Vec<ExprIdx>>,
```

Comment:
No need to `Box` if you have a `Vec` here. The boxing was only needed because you cannot store a slice directly